core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -480,11 +481,15 @@ var ExplorationEditorMainTab = function() {
   // 'richTextInstructions' is a function that is sent a RichTextEditor which it
   // can then use to alter the state content, for example by calling
   // .appendBoldText(...).
-  this.setContent = async function(richTextInstructions) {
+  this.setContent = async function(richTextInstructions, expectFadeIn) {
```

Comment:
Maybe have `expectFadeIn` default to `false`? I think that will be more readable than having it default to undefined

Reply:
Done.